src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -468,11 +468,16 @@ internal MirrorData GetCachedValueFromEngine(EngineController engine)
         }
 
         /// <summary>
-        /// Use to indicated if a node was involved in the most recent graph evaluation.
+        /// Use to indicate if a node was involved in the most recent graph evaluation.
         /// </summary>
         internal bool WasInvolvedInExecution { get; set; }
 
         /// <summary>
+        /// Used to indicate wheter the render package of a node has been updated after last execution.
+        /// </summary>
+        internal bool WasRenderPackageUpdatedAfterExecution { get; set; }
```

Comment:
I think we can say this instead:

```csharp
/// This flag indicates if new render packages of a NodeModel has been updated 
/// since the last execution. UpdateRenderPackageAsyncTask will always be 
/// generated for a NodeModel that took part in the evaluation, if this flag 
/// is false.
```